explainer/explainer11.md

```diff
@@ -1,49 +1,52 @@
 
 The W3C Web of Things (WoT) was created to counter the fragmentation of the IoT market:
 The variety of pre-existing standards and vendor-specific platforms in the various verticals (i.e., application domains) has led to data and application silos and many difficult system integration problems.
 By some estimates, [up to 40% of the value of the IoT](https://www.mckinsey.com/business-functions/digital-mckinsey/our-insights/the-internet-of-things-the-value-of-digitizing-the-physical-world) is only acheivable through interoperabilty, which is the primary goal of the Web of Things.
 
 W3C WoT acknowledges that the IoT emerged from and incorporates various established ecosystems with existing platforms and standards that reflect the diversified requirements of IoT applications (e.g., network protocols, data models).
-Thus, W3C WoT uses Web technology to **formally describe the interfaces** of IoT devices and platforms to ease the integration across ecosystems and application domains.
-The idea is that rich metadata based on URIs, hypermedia controls, and collaborative schema definitions (cf. [schema.org]()) allows clients to adapt to the peculiarities of a given IoT platform by configuring its (vanilla) protocol stack with the metadata (e.g., only one CoAP stack for OCF, oneM2M, LwM2M, and dotdot -- or re-using the same HTTP API instead of generating custom stubs for each "RESTful API").
+Thus, rather than attempting to establish yet another competing ecosystem,
+W3C WoT uses Web technology to **formally describe the interfaces** of IoT devices and platforms, including those in existing ecosystems, to ease the integration across ecosystems and application domains.
+The idea is that rich metadata based on URIs, hypermedia controls, and collaborative schema definitions (cf. [schema.org](https://schema.org)) allows clients to adapt to the peculiarities of a given IoT platform by configuring its (vanilla) protocol stack with appropriate metadata (e.g., enabling the use of only one CoAP stack for OCF, oneM2M, LwM2M, and dotdot -- or re-using the same HTTP API instead of generating custom stubs for each "RESTful API").
 
-The WoT Architecture document serves as an umbrella for all WoT specifications, and hence also sets the general goals of the W3C Web of Things.
-It defines the abstract architecture and spans the design space for current and future so-called building blocks, which are specified in separate documents.
-Specifically, the purpose of the WoT Architecture document is to provide
+The WoT Architecture document serves as an umbrella for all WoT specifications, and sets the general goals of the W3C Web of Things.
+It defines the abstract architecture and the design space for current and future building blocks, each of which are specified in separate documents.
+Specifically, the purpose of the WoT Architecture document is to provide:
+- basic terminology definitions;
+- use cases and derived requirements the W3C WoT Architecture should satisfy;
```

Comment:
please remove this line.

Reply:
which line?  line 16?  Unfortunately github is being vague and is showing me four lines.   I assume the last one about use cases and derived requirements?

Reply:
The use cases and requirements were moved to the usecases document, so we should delete line 16.